### PR TITLE
[chore] updated versions

### PR DESCRIPTION
**Pull request by gcnyin**
**Summary of changes**
Changes introduced in this pull request:
- 

- jdk 20.0.1_9
- node 20
- quarkus 3.x
- zio-grpc 0.6.0-rc5

**Reference issue to close (if applicable)**

**Other information and links**

### CODE REVIEW

File: scala_zio_bench/src/main/scala/com/example/helloworld/GreeterServer.scala [hunk 1]
```diff
@@ -2,42 +2,8 @@ package com.example.helloworld
...
-
-    val value = System.getenv.getOrDefault("JVM_EXECUTOR_TYPE", "workStealing")
-    value match {
-      case "direct" => sb.directExecutor
-      case "single" => sb.executor(Executors.newSingleThreadExecutor)
-      case "fixed" => sb.executor(Executors.newFixedThreadPool(i_threads))
-      case "workStealing" => sb.executor(Executors.newWorkStealingPool(i_threads))
-      case "cached" => sb.executor(Executors.newCachedThreadPool)
-    }
-    sb
```
Comment (LesnyRumcajs) [inline scala_zio_bench/src/main/scala/com/example/helloworld/GreeterServer.scala:39]: Why did you remove this logic?

Comment (LesnyRumcajs) [reply scala_zio_bench/src/main/scala/com/example/helloworld/GreeterServer.scala:39]: That is surprising they wouldn't add an alternative approach. Could you please point me to a relevant changelog entry?

Comment (gcnyin) [reply scala_zio_bench/src/main/scala/com/example/helloworld/GreeterServer.scala:39]: It's hard to get details from public information.

In my opinion, `zio-grpc` api is not stable, the authors changed it many times. And this builder configuration will not have a great impact on the test results.

Comment (LesnyRumcajs) [reply scala_zio_bench/src/main/scala/com/example/helloworld/GreeterServer.scala:39]: Alright, thanks.

**Review by LesnyRumcajs — APPROVED** [review]

**Comment by LesnyRumcajs** [comment]
@gcnyin thank you!

**Comment by omidb** [comment]
@LesnyRumcajs and @gcnyin have you run the benchmark yet?

**Comment by LesnyRumcajs** [comment]
@omidb No. I can do it this weekend if I have time on my new desktop.

**Comment by LesnyRumcajs** [comment]
> @LesnyRumcajs and @gcnyin have you run the benchmark yet?

https://github.com/LesnyRumcajs/grpc_bench/discussions/354
